requirements.txt

```diff
@@ -5,9 +5,11 @@ chardet==3.0.4
 click==7.0
 coloredlogs==10.0
 cryptography==2.8
+dateutil==2.8.1
 docutils==0.15.2
 github3.py==1.3.0
 humanfriendly==4.18
+fastnumbers==2.2.1
```

Comment:
It actually does. The system tries to cast every single string as a number. I identified my attempt as a bottleneck. Maybe there would have been a faster technique. I was using a try/catch block (actually one for floats and one for ints) instead of a regexp-and-then-convert-if-matches. If you feel strongly I could benchmark a regexp-based solution.

This will soon become a dependency of Snowfakery rather than CCI but until we have plugins it would also be a transitive dependency of CCI.